### ISSUE TITLE
Add booking info to analytics events 

### ISSUE THREAD

**Issue by git-toni** [kiwicom/smart-faq#465]
Ideally we should send booking info together with our tracking events. Example with the main event `smartFAQ` with clickOnHelp/close actions

```
eventName: "smartFAQ"
attributes: {
  action: "clickOnHelp"
  booking: true / false,
  urgency: high / normal / afterFlight
}
```
We retrieve flight info from GraphQL at `ContentPage` component's `QueryRenderer`, so being able to send this info from the very first moment one opens up smart-faq might involve a painful refactor. 
Imho it's worth thinking if there's a way around it ( maybe keeping a flag in context about whether the open event has been sent or not until the `QueryRenderer` is resolved and we have the upcoming flight info?).

**Comment by jalbertsr** [comment]
closing in favour of https://github.com/kiwicom/smart-faq/issues/637
